docs/userguide.md

```diff
@@ -69,6 +69,18 @@ factoryResource.use { factory =>
 ## Extensions
 Extensions are packages that monitor some functionality for you - without you having to do anything.
 
+### Http4s
+Http4s package (`datadog-scala-http4s`) provides implementation of [MetricsOps](https://http4s.org/v0.20/api/org/http4s/metrics/metricsops) that is used by [http4s](https://http4s.org) to report both client and server metrics.
+
+```scala mdoc
+import com.avast.cloud.metrics.datadog.extension.http4s._
```

Comment:
pls make the example more similar to the JVM example - either update that one but the usage is very similar so lets  make examples similar ... pick whichever style feels more understandable to you

Reply:
I'm not really sure what you are referring to.

Reply:
Nvm... My mistake